plugin/ochttp/trace.go

```diff
@@ -146,7 +146,7 @@ func responseAttrs(resp *http.Response) []trace.Attribute {
 	}
 }
 
-// TraceStatus converts the HTTP status code to a trace.Status that
+// TraceStatus is a utility to convert the HTTP status code to a trace.Status that
```

Comment:
TraceStatus converts the HTTP status code.

Whether it is a utility or not is irrelevant.

Reply:
Looking through docs, it's unclear why this is there. My saying it's a utility, I intend to deflect attention off of them function as something that most users won't be interested in. Do you want to suggest another way of saying this?